src/middle/Semantic_error.ml

```diff
@@ -168,6 +169,13 @@ module TypeError = struct
           "A returning function was expected but an undeclared identifier \
            '%s' was supplied."
           fn_name
+    | ReturningFnExpectedUndeclaredDistSuffixFound (prefix, suffix) ->
+        Fmt.pf ppf
+          "A function was expected but an unknown identifier %s was recieved. \
+           This appears to be part of the %s family of distributions, for \
+           which the %s suffix in not implemented."
```

Comment:
@bob-carpenter @betanalpha or anyone else with more experience working with end-users. What would be the best way to describe this error of a missing implementation for a function of a distribution? Not saying this one is not good, just wanted to get more input.

Reply:
It's not really an unknown identifier when it matches the expected pattern. How about simply
```ocaml
"Function '%s_%s' is not implemented for distribution '%s'." prefix suffix prefix
```